client/preprocessors.py

```diff
@@ -32,27 +34,28 @@ def __init__(self, rc, source_dir, output_dir):
         self.source_dir = source_dir  # Local directory
         self.output_dir = output_dir  # Local directory
 
-        # Copy all files in the root of source_dir
-        for file_path in glob(self.source_dir+"/*"):
-            if os.path.isfile(file_path):
-                copy(file_path, self.output_dir)
-
         # Write out the new manifest file based on the resource container
         write_file(os.path.join(self.output_dir, 'manifest.json'), self.rc.as_dict())
+        write_file(os.path.join(self.output_dir, 'manifest.yaml'), self.rc.as_dict())
```

Comment:
Yeah, that was intentional, just because it's always included a generated manifest.json file, but now we seem to want manifest.yaml file, per this comment on the download feature (https://github.com/unfoldingWord-dev/door43.org/issues/71#issuecomment-304093697). I could be wrong, but saw .json better for machines to read (loading in JavaScript, etc.) and YAML for humans, but we probably won't need to use the .json file on the live.door43.org pages, so will stop serving the manifest.json file. Just writing that here for future reference.